ietf/static/js/timezone.js

```diff
@@ -1,5 +1,5 @@
-        // Copyright The IETF Trust 2021, All Rights Reserved
-
+// Copyright The IETF Trust 2021, All Rights Reserved
+/* global moment */
```

Comment:
What does this do?

Reply:
It notifies eslint that 'moment' is a globally defined symbol

Reply:
(though it certainly looks like something that we should all share on Earth Day)